File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int ans = 0 ; 
        int n = nums.length;
        for(int i = 1 ;i<=n;i++) ans = ans^i;
        for(int i = 0 ;i<nums.length ;i++){
            ans = ans^nums[i];
        }
        return ans;
    }
}

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        ans = 0
        i = 0
        n = len(nums) 
        while i<=n:
            ans = ans^i
            i+=1

        for i in nums:
            ans = ans^i
        return ans
        